apps/system/js/statusbar.js

```diff
@@ -194,7 +194,6 @@ var StatusBar = {
     window.addEventListener('appopened', this);
     window.addEventListener('hierarchytopmostwindowchanged', this);
     window.addEventListener('activityopened', this);
-    window.addEventListener('activityterminated', this);
     window.addEventListener('activitydestroyed', this);
```

Comment:
We can't unfortunately because we still need to `updateMinimizedStatusBarWidth` here to prevent bug 1088462.